public/configs/v1/env.json

```diff
@@ -157,7 +157,8 @@
          "protocolStaking": "[HTTP link to protocol staking info]",
          "stakingAndClaimingRewardsLearnMore": "[HTTP link to staking and claiming rewards learn more]",
          "predictionMarketLearnMore": "[HTTP link to prediction market learn more]",
-         "discoveryProgram": "[HTTP link to discovery program learn more]"
+         "discoveryProgram": "[HTTP link to discovery program learn more]",
+				 "top100TradersGetInTouch": "[HTTP link to get in touch with 100 traders program]"
```

Comment:
[q] should i be adding this in the links for other envs? i didn't want to suggest a value since this is probably unique per deployer, it's not like there's a default suggested one deployers should use

Reply:
eh I think i can just populate it with an empty string. Not really necessary for the other envs, as long as it doesn't crash on click for testing